lib/ios-deploy.js

```diff
@@ -52,6 +52,10 @@ class IOSDeploy {
       return false;
     }
   }
+
```

Comment:
it would be, probably, easier to read if we add a comment about this method is an alias
It's a pity, that ES6 does not have built-in syntax sugar for aliases creation like python or ruby %)